### PR TITLE
[SPARK-29571][SQL][TESTS][FOLLOWUP] Fix UT in AllExecutionsPageSuite

### PR DESCRIPTION
**Pull request by 07ARB**
### What changes were proposed in this pull request?

This is a follow-up of #24052 to correct assert condition.

### Why are the changes needed?
 To test IllegalArgumentException condition..

### Does this PR introduce any user-facing change?
 No.

### How was this patch tested?

Manual Test (during fixing of SPARK-29453 find this issue)

### CODE REVIEW

**Review by HeartSaVioR — APPROVED** [review]
LGTM

**Comment by 07ARB** [comment]
@HeartSaVioR , Please check this two statement in AllExecutionsPageSuite.scala Testname : "sorting should be successful"
val html = renderSQLPage(request, tab, statusStore).toString().toLowerCase(Locale.ROOT)
assert(!html.contains("IllegalArgumentException"))

**Comment by HeartSaVioR** [comment]
Yeah I know what this patch is fixing, so left +1 (non-binding). The change won't make existing test fail, right?

**Comment by 07ARB** [comment]
@HeartSaVioR , if it's ok then please review my PR.

**Comment by 07ARB** [comment]
> Yeah I know what this patch is fixing, so left +1 (non-binding). The change won't make existing test fail, right?

yes

**Comment by wangyum** [comment]
cc @shahidki31

**Comment by wangyum** [comment]
ok to test

**Comment by dongjoon-hyun** [comment]
Hi, @07ARB . Thank you for splitting this.
I update the PR description style properly.

**Review by dongjoon-hyun — APPROVED** [review]

**Review by shahidki31 — APPROVED** [review]

**Comment by 07ARB** [comment]
> LGTM

@HeartSaVioR, @dongjoon-hyun ,@shahidki31 Thank you

**Review by maropu — APPROVED** [review]

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

File: sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ class AllExecutionsPageSuite extends SharedSparkSession with BeforeAndAfter {
     map.put("failed.sort", Array("duration"))
     when(request.getParameterMap()).thenReturn(map)
     val html = renderSQLPage(request, tab, statusStore).toString().toLowerCase(Locale.ROOT)
-    assert(!html.contains("IllegalArgumentException"))
+    assert(!html.contains("illegalargumentexception"))
```
[the remark this answers is not in the document]
Comment (HeartSaVioR) [reply sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala]: So the problem was that the assertion effectively doesn't verify anything - finding upper&lower string from lowercased string.

**Pull request closed without merging**